src/Algos.cc

```diff
@@ -39,48 +40,23 @@ toMacroName(const std::string_view name) noexcept {
 }
 
 int
-execCmd(const std::string_view cmd) noexcept {
+execCmd(const Command& cmd) noexcept {
   logger::debug("Running `", cmd, '`');
-  const int status = std::system(cmd.data());
-  const int exitCode = status >> 8;
-  return exitCode;
-}
-
-static std::pair<std::string, int>
-getCmdOutputImpl(const std::string_view cmd) {
-  constexpr usize bufferSize = 128;
-  std::array<char, bufferSize> buffer{};
-  std::string output;
-
-  FILE* pipe = popen(cmd.data(), "r");
-  if (!pipe) {
-    throw PoacError("popen() failed!");
-  }
-
-  while (fgets(buffer.data(), buffer.size(), pipe) != nullptr) {
-    output += buffer.data();
-  }
-
-  const int status = pclose(pipe);
-  if (status == -1) {
-    throw PoacError("pclose() failed!");
-  }
-  const int exitCode = status >> 8;
-  return { output, exitCode };
+  return cmd.output().exitCode;
```

Comment:
When we use `execCmd`,  we would rather not capture the output. We may want to implement [`Command::spawn`](https://doc.rust-lang.org/std/process/struct.Command.html#method.spawn) instead.